app/controllers/api/concerns/errors.rb

```diff
@@ -0,0 +1,77 @@
+module Api::Concerns::Errors
```

Comment:
All this fanfare around overriding Rails' error handling seems kinda redundant, we're basically just restating the status code and some other obvious information throughout 5 additional files.

Reply:
Notes from meeting:
- We chatted about how Rails doesn't provide the best error handling for JSON responses
- This code handles serializing error objects into JSON responses, which is something Rails doesn't provide when we add additional fields such as `title` and `detail`.
- For more information regarding JSON error responses, see https://jsonapi.org/format/#errors